3d_font.txt

```diff
@@ -433,3 +433,31 @@ z
         | \________/  /
         \ |        | /
          \|________|/
+4
+ ____       ____
+|\   \     |\   \
+| \   \    | \   \
+\  \   \___\__\   \
+ \  \              \
+  \  \___________   \
+   \ |          |\   \
+    \|__________| \   \
+                \  \   \
+                 \  \   \
+                  \  \___\
+                   \ |   |
+                    \|___|
+5
+   ___________________
+  /                   \
+ |\    ________________\
+ | \   \               |
+ \  \   \______________|__
+  \  \                    \
+   \  \_________________   \
+    \ |                /   /|
+     \|__ ____________/   / |
```

Comment:
This slant at the edge is 4 lines long, which isn't matching with other characters. Try reducing it to 3 lines